students/cbrown/Lesson 9/mailroom_oo_final.py

```diff
@@ -0,0 +1,224 @@
+import sys
+
+
+class Donor_Collection():
+
+    donor_dict = {"Bill Gates": [539000, 235642],
+          "Jeff Bezos": [108356, 204295, 897345],
+          "Satya Nadella": [236000, 305352],
+          "Mark Zuckerberg": [153956.35],
+          "Mark Cuban":[459035, 369.50, 570.89]}
+
+    def __init__(self, donor=None):
+        self.donor = donor
```

Comment:
the relationship here is incorrect. DonorCollection should reference a collection of donor objects.

self.donor represents a single Donor object which is not a valid relationship.